fedgov-cv-TIMcreator-webapp/src/main/webapp/js/mapping.js

```diff
@@ -697,6 +681,8 @@ function registerMapEvents() {
   vectors = new ol.layer.Vector({
     source: vectorSource,
     style: vectorStyle,

```

Comment:
The vectorStyle inside the style.js is the same as  the removed vectoryStyle. Shouldn't we use the icon instead of the marker? Refer to: https://github.com/usdot-fhwa-stol/connectedvcs-tools/blob/e45620da8a1246040a9d09d2cd8f9c9c77c32fc6/fedgov-cv-ISDcreator-webapp/src/main/webapp/js/interactions.js#L395